src/backend/libs/efiling-mail-it/src/main/java/ca/bc/gov/open/jag/jagmailit/MailSendProperties.java

```diff
@@ -0,0 +1,31 @@
+package ca.bc.gov.open.jag.jagmailit;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "mailsend")
+public class MailSendProperties {
+
+	private String hostName;
+	private Integer port;
+
+	public String getHostName() {
+		return hostName;
```

Comment:
I think we should request for a baseUrl, we should refactor the rest. hostname does not imply that you specify the http scheme, baseUrl does and it can include the port as well. what do you think ?

Reply:
I agree.  I made this change yesterday.  Committed now.